server/routes/api.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express')
+const router = express.Router()
+const urllib = require('urllib')
+
+const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756",
+}
+
+let dreamTeam = []
+
+router.get(`/teams/:teamName`, function (req, res) {
+    urllib.request('http://data.nba.net/10s/prod/v1/2018/players.json', function (err, playerData) {
+        if (err) {
+            console.log("error, try again")
+            throw err
+        }
+        let playersArr = []
+        let players = JSON.parse(playerData.toString()).league.standard
+        let teamIds = teamToIDs[req.params.teamName]
+        for (let player of players) {
+            if (player.teamId === teamIds & player.isActive) {
+                playersArr.push(player)
+            }
+        }
+        res.send(playersArr)
+    })
+})
+router.get(`/playerStats/:last/:first`, function (req, res) {
+    let playerFirst = req.params.first
+    let playerLast = req.params.last
+    urllib.request(`https://nba-players.herokuapp.com/players-stats/${playerLast}/${playerFirst}`, function (err, playerStats) {
+
+        try {
```

Comment:
you don't need try-catch here. if you will get an error, the "err" parameter will not be undefined -> then you can just use if statement to ask if there is an error or not